### ISSUE TITLE
Menus don't work in every DVD I've tried so far

### ISSUE THREAD

**Issue by Blamsoft** [gmarty/DVD.js#12]
You have a nice, beautiful, priceless shelf of DVDs! Did you test any of them with DVD.js?? Do you know how to rip a protected DVD???

**Comment by Blamsoft** [comment]
dvdbackup -M --input=/dev/sr0 --output=. --name="LOTR: The Fellowship of the Ring" --progress --verbose

**Comment by Blamsoft** [comment]
https://manpages.org/dvdbackup

**Comment by gmarty** [comment]
Yes, so this project is a proof of concept more than anything else. It's not meant to be a fully fledged DVD player.
For menus specifically, there is a lot of work that needs to be done on top of implementing all the opcodes (e.g. generating the buttons using canvas and the palettes...).

**Comment by Blamsoft** [comment]
No, this project is not meant to be a proof of concept. It’s meant to be a full fledged DVD player. Did you get the DVD specification you mentioned or do you want me to purchase it?
